projects/angular/src/wizard/_wizard.clarity.scss

```diff
@@ -25,129 +27,52 @@
-      );
+      box-shadow: wizard-variables.$clr-wizard-box-shadow;
       height: 50%;
       max-height: 100%;
     }
 
     .modal-content {
-      border-radius: 0;
-      @include mixins.css-var(
-        border-top-right-radius,
-        clr-wizard-border-radius,
-        wizard-variables.$clr-wizard-border-radius,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        border-bottom-right-radius,
-        clr-wizard-border-radius,
-        wizard-variables.$clr-wizard-border-radius,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        background-color,
-        clr-wizard-content-background-color,
-        modal-variables.$clr-modal-bg-color,
-        variables.$clr-use-custom-properties
-      );
+      border-radius: 0 wizard-variables.$clr-wizard-border-radius wizard-variables.$clr-wizard-border-radius 0;
       box-shadow: none;
-      padding: 0;
+      padding: wizard-variables.$clr-wizard-default-space;
       flex: 2 2 auto;
       width: 66%;
       height: initial;
       overflow: hidden;
       display: flex;
       align-items: flex-start;
       flex-direction: column;
+      gap: wizard-variables.$clr-wizard-default-space;
     }
 
     .modal-header--accessible {
       flex: 0 0 auto;
       width: 100%;
-      // right padding nudged a bit to better align icons
-      @include mixins.css-var(
-        padding,
-        clr-wizard-header-padding,
-        wizard-variables.$clr-wizard-default-space
-          (wizard-variables.$clr-wizard-default-space - variables.$clr_baselineRem_5px) variables.$clr_baselineRem_0_25
-          wizard-variables.$clr-wizard-default-space,
-        variables.$clr-use-custom-properties
-      );
+      padding: 0;
     }
 
     .modal-title {
-      @include mixins.css-var(
-        color,
-        clr-wizard-title-text,
-        wizard-variables.$clr-wizard-title-text,
-        variables.$clr-use-custom-properties
-      );
+      color: wizard-variables.$clr-wizard-title-text;
       display: flex;
       flex-direction: row;
       width: 100%;
-      // align focus style across modal title
-      @include mixins.css-var(
-        font-size,
-        clr-h4-font-size,
-        modal-variables.$clr-modal-title-font-size,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        line-height,
-        clr-h4-line-height,
-        variables.$clr_baselineRem_1_13,
-        variables.$clr-use-custom-properties
-      );
-      margin-top: -1 * variables.$clr_baselineRem_0_13;
     }
 
     .modal-body {
       flex: 1 1 auto;
-      @include mixins.css-var(
-        color,
-        clr-wizard-main-text-color,
-        wizard-variables.$clr-wizard-main-text-color,
-        variables.$clr-use-custom-properties
-      );
+      color: wizard-variables.$clr-wizard-main-text-color;
       width: 100%;
     }
 
-    .modal-footer {
+    .modal-footer.clr-wizard-footer {
```

Comment:
This is already under `clr-wizard` so except for more specificity is that needed?

Reply:
removed in commit b19c78a